Project/JSONAPITests/JSONAPITests.m

```diff
@@ -239,8 +239,27 @@ - (void)testGenericSerialization {
     XCTAssertEqualObjects(serializedSecondPost[@"relationships"][@"publisher"][@"data"][@"type"], @"SocialCommunity", @"SocialCommunity type should be 'SocialCommunity'");
 }
 
+#pragma mark - Empty relationship tests
+
+- (void)testEmptyRelationship {
+    NSDictionary *json = [self emptyRelationshipsExampleJSON];
+    JSONAPI *jsonAPI = [JSONAPI jsonAPIWithDictionary:json];
+    
+    NewsFeedPostResource *testPost = jsonAPI.resource;
+    
+    XCTAssertNil(testPost.publisher, @"Test post's attachments should not be nil");
```

Comment:
Small copy-and-paste error: the description should read `Test post's publisher should be nil`.